flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/ContinuousFileReaderOperator.java

```diff
@@ -80,6 +81,7 @@
 
 	public ContinuousFileReaderOperator(FileInputFormat<OUT> format) {
 		this.format = checkNotNull(format);
+		setChainingStrategy(ChainingStrategy.NEVER);
```

Comment:
I think this should be mentioned in the release notes on this branch (that since 1.10.X, we disabled chaining because of yadayadayada).